src/os/inc/osapi-file.h

```diff
@@ -382,7 +382,7 @@ int32 OS_remove(const char *path);
  * @retval #OS_FS_ERR_PATH_TOO_LONG if the paths given are too long to be stored locally
  * @retval #OS_FS_ERR_NAME_TOO_LONG if the new name is too long to be stored locally
  */
-int32 OS_rename(const char *old_filename, const char *new_filename);
+int32 OS_rename(const char *old, const char *new);
```

Comment:
We should strive to avoid changing the name in the header/declaration, but rather prefer to update the implementation to match the declaration whenever possible.

Note that in this particular case `new` is a reserved keyword in C++ but not reserved in C.  There are some C++ users that would get tripped up if the name were changed to `new`.

Reply:
Good point re. the reserved keyword.
Will update this and the rest of the implementations to match the prototypes.